observation_portal/requestgroups/test/test_api.py

```diff
@@ -2825,11 +2830,47 @@ def test_last_change_date_is_updated_when_request_is_submitted(self):
         last_change = response.json()['last_change_time']
         self.assertAlmostEqual(datetime_parser(last_change), timezone.now(), delta=timedelta(minutes=1))
 
+    def test_last_change_date_telescope_class_is_updated_when_request_is_submitted(self):
+        last_change_cached = self.locmem_cache.get('observation_portal_last_change_time_1m0')
+        self.assertIsNone(last_change_cached)
+        rg = self.generic_payload.copy()
+        response = self.client.post(reverse('api:request_groups-list'), data=self.generic_payload)
+
+        response = self.client.get(reverse('api:last_changed') + '?telescope_class=1m0')
+        last_change = response.json()['last_change_time']
+        self.assertAlmostEqual(datetime_parser(last_change), timezone.now(), delta=timedelta(minutes=1))
```

Comment:
I guess saving the time before posting the request and then using an assertGreater with that would work too. This just uses assertAlmostEqual within a minute because timezone.now() will obviously be a little ahead, but just microseconds (Assuming posting the request just updated the time), so it will be almost equal.

Reply:
Gotcha. I think comparing to a time right before posting the request would be a bit more clear in the intention of the test.